oap-server/server-starter/src/main/resources/ui-initialized-templates/mongodb/mongodb-cluster.json

```diff
@@ -231,7 +232,7 @@
                     "tableHeaderCol2": ""
                   },
                   "expressions": [
-                    "meter_mongodb_cluster_db_data_size/1024/1024/1024"
+                    "aggregate_labels(meter_mongodb_cluster_db_data_size, sum(database))/1024/1024/1024"
```

Comment:
suggest using the `latest` or `avg`... function to aggregate values if the widget can't show the time series values.

Reply:
Updated.